openage/testing/main.py

```diff
@@ -138,3 +144,8 @@ def main(args, error):
     if args.demo:
         _, _, _, demofun = test_list[args.demo[0], 'demo']
         exit(demofun(args.demo[1:]))
+
+    if args.benchmark:
+        from openage.testing.benchmark import benchmark_func
```

Comment:
why not import ealier at top-level? and it should be `from .benchmark import benchmark_func`
The function to perform the benchmark should be called `benchmark` i'd say, the testing function `benchmark_test`, then the naming of the "relevant" function would be better.